examples/nidcpower_source_dc_voltage/measurement.py

```diff
@@ -72,7 +72,7 @@ def measure(
                 context=measurement_service.context.pin_map_context,
                 pin_or_relay_names=pin_names,
                 instrument_type_id=nims.session_management.INSTRUMENT_TYPE_NI_DCPOWER,
-                timeout=-1,
+                timeout=60000,
```

Comment:
In Python, timeouts are specified in seconds, so use 60.

Please test one of the examples in both InstrumentStudio and TestStand, including the timeout case.

Reply:
FYI: the type annotation says `Optional[float]` and `60` is an `int`, but that's ok: https://mypy.readthedocs.io/en/latest/duck_type_compatibility.html

Reply:
Any good ideas about how to test the timeout case? My thought was to build in a 60 second delay in the session manager service in my local build before it checks the timeout so it would always time out.

I figured if I just killed or made the session manager service missing, it would error out before it had a chance to timeout during the reserve session call.

Reply:
Run another measurement service that uses DCPower. Add a delay to that service or attach a debugger to it.

Reply:
Tested with delay in InstrumentStudio and TestStand. InstrumentStudio - errors out after the specified delay (60 seconds). TestStand - errors out immediately.